contracts/FlowCallbackScheduler.cdc

```diff
@@ -554,20 +592,42 @@ access(all) contract FlowCallbackScheduler {
                     return false
                 }
 
-                return timestamp <= currentTimestamp 
+                return timestamp <= currentTimestamp
             }
             let pastTimestamps = self.slotQueue.keys.filter(pastTimestamp)
             
             // process all callbacks from timestamps in the past
+            // and add low priority callbacks to the timestamp if there is space
             for timestamp in pastTimestamps {
-                let callbackIDs = self.slotQueue[timestamp] ?? []
+                let callbackIDs = self.slotQueue[timestamp] ?? {}
+
+                // Add low priority callbacks to the list
+                // until the low available effort is used up
+                // todo: This could get pretty costly if there are a lot of low priority callbacks
+                // in the queue. Figure out how to more efficiently go through the low priority callbacks
+                // Could potentially limit the size of the low priority callback queue?
```

Comment:
The low priority queue is limited with effort limit, so each timestamp will only have max X low priorities. Not sure what you mean with this?